js/src/wrappers/test-prompts.ts

```diff
@@ -0,0 +1,144 @@
+// Very long system prompt to trigger Anthropic prompt caching (1500+ tokens)
```

Comment:
#nit i usually do something like `ai-sdk-v2.fixtures.ts` to point out that it's not an interesting file plus which source code depends on it. not a standard thing, though